content/4_Gold/Shortest_Paths.mdx

```diff
@@ -169,6 +169,44 @@ public class Main {
 
 </JavaSection>
 
+<PySection>
+
+```py
+n, m, q = map(int, input().split())
+
+mat = [[int(1e18)] * n for _ in range(n)]
```

Comment:
also inf here

Reply:
putting inf here makes the code run slower (tles on 9 test cases rather than only 2)

should i still change it?

Reply:
oh then it's fine ig